actionpack/CHANGELOG.md

```diff
@@ -140,4 +140,42 @@
 
     *Petrik de Heus*
 
+*   Add comprehensive support for HTTP Cache-Control request directives according to RFC 9111.
+
+    Provides a `request.cache_control_directives` object that gives access to request cache directives:
+
+    ```ruby
+    # Boolean directives
+    request.cache_control_directives.only_if_cached?  # => true/false
+    request.cache_control_directives.no_cache?        # => true/false
+    request.cache_control_directives.no_store?        # => true/false
+    request.cache_control_directives.no_transform?    # => true/false
+
+    # Value directives
+    request.cache_control_directives.max_age          # => integer or nil
+    request.cache_control_directives.max_stale        # => integer or nil (or true for valueless max-stale)
```

Comment:
This is a bit of a problem, users are likely to assume `max_stale` if truthy is an Integer and not handle `true` correct.

Reply:
Nevermind, I see `max_stale_unlimited?`. I suppose that is acceptable